gcampus/analysis/views.py

```diff
@@ -1,11 +1,32 @@
+import math
+
 from django.utils.translation import gettext_lazy as _
-from django.views.generic import ListView
+from django.views.generic import ListView, DetailView
 
-from gcampus.core.models import ParameterType
+from gcampus.core.models import ParameterType, Calibration
 from gcampus.core.views.base import TitleMixin
 
 
-class ODConverterView(TitleMixin, ListView):
+class ODConverterOverView(TitleMixin, ListView):
     title = _("Optical Density Converter")
-    template_name = "gcampusanalysis/od_converter.html"
+    template_name = "gcampusanalysis/od_converter_overview.html"
     model = ParameterType
+    context_object_name = "parameter"
+
+    def get_context_data(self, **kwargs):
+        kwargs["row_num"] = math.ceil(len(ParameterType.objects.all()) / 3)
```

Comment:
Instead of `ParameterType.objects.all()`, you can use `self.object_list`. Calling `len(ParameterType.objects.all())` will query all parameter types from the database. However, this has already been done at an earlier point and the parameter types are now available under `self.object_list`. This will remove two database queries :rocket:.

Reply:
fixed in 72a9f26